packages/example-react/project/app/index.js

```diff
@@ -5,4 +5,4 @@ import './index.scss';
 import './index.css';
 import App from './App';
 
-render(<App />, document.getElementById('root'));
+render(<App />, document.querySelector('root'));
```

Comment:
Probably want `'#root'` to select the element with the id of root.

Reply:
Yeah saw that typo as well. Removing the eslint-config-availity plugin out of the package as well since its already inside the workflow-plugin-react package

Reply:
https://github.com/Availity/availity-workflow/commit/8c4a5e42c9577573c1efb390c6be540d158d2dcd#diff-bf75262b5b8b152128c0e012aa770127L8